File: app/src/main/java/com/senerunosoft/puantablosu/model/Player.java
package com.senerunosoft.puantablosu.model;

import java.util.ArrayList;
import java.util.List;
import java.util.UUID;

public class Player {

    private String id;
    private String name;

    public Player() {
    }

    public Player(String name) {
        this.id = UUID.randomUUID().toString();
        this.name = name;
    }

    public String getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

}

File: app/src/main/java/com/senerunosoft/puantablosu/model/Player.kt
package com.senerunosoft.puantablosu.model

import java.util.UUID

data class Player(
    val id: String = UUID.randomUUID().toString(),
    val name: String = ""
) {
    // Empty constructor for Gson
    constructor() : this(UUID.randomUUID().toString(), "")
    
    // Constructor matching the original Java constructor
    constructor(name: String) : this(UUID.randomUUID().toString(), name)
}